configure.ac

```diff
@@ -74,6 +79,15 @@ AC_CHECK_PROG(AR,ar,ar)
 AC_CHECK_PROG(STRIP,strip,strip)
 AC_CHECK_PROG(SED,sed,sed)
 
+if test $glpk = "true"; then
+    dnl check for glpk
+    AC_CHECK_HEADER([glpk.h],[AC_DEFINE([GLPK],[1],[glpk solver for solving ILP])]
+                                CFLAGS="$CFLAGS -DGLPK" LDFLAGS="$LDFLAGS -lglpk"
+                                , [AC_ERROR([glpk.h header not found])])
+    AC_CHECK_LIB([glpk], [glp_create_prob], []
+                 , [AC_ERROR([glpk library not found])])
+fi
+
```

Comment:
You also need to check for a minimum GLPK version. Versions much older than recent ones may not work.